### ISSUE TITLE
Local test case not updated after kata updated.

### ISSUE THREAD

**Issue by kuchaguangjie** [codewars/codewars.com#2544]
**Describe the bug**
I created a kata, in Java, and trained the kata, then I updated the test case, modified the class name.
Then when I train the kata again, the test case is not updated, will cause error, since class name changed.

**Expected behavior**
After kata updated & published, when train the kata again, I should get the updated test cases.

**Browser**
Chrome 94.0.4606.81 (Official Build) (64-bit)

- OS: Linux
- Browser: chrome
- Version: 94.0.4606.81 (Official Build) (64-bit)

**Comment by Blind4Basics** [comment]
you need to refresh the trainer.

**Comment by kazk** [comment]
This is intentional so we don't overwrite anything changed by the user. You can always reset to get the latest tests.

**Comment by kuchaguangjie** [comment]
Ok, or you may hint user to refresh to get updated code / test, if you can detect there is a change.

**Comment by kazk** [comment]
Yeah, we can check if the user started before the last publish, and show something. But I don't think we can currently tell if the last publish was relevant to the language the user is using. Showing something based on the difference will have too many false positives.

I'll mark this an enhancement.
